tap-snapshots/src/node-benchmark-compare.test.js.test.cjs

```diff
@@ -123,6 +136,18 @@ In this case, there are 108 comparisons, you can thus expect the following amoun
   0.11 false positives, when considering a 0.1% risk acceptance (***)
 `
 
+exports[`src/node-benchmark-compare.test.js TAP insufficient > must match snapshot 1`] = `
+                                                                   confidence improvement accuracy (*) (**) (***)
+http/client-request-body.js method='write' len=32 type='asc' dur=5         NA  Infinity %           NA   NA    NA
+http/something.js method='write' len=32 type='asc' dur=5                   NA       NaN %           NA   NA    NA
```

Comment:
Shouldn't it be `NA` instead of `Infinity`/`NaN`?

Reply:
It don't know. Ideally should be the same as with the R script

Reply:
It is.
```
                                                                   confidence improvement accuracy (*) (**) (***)
http/client-request-body.js method='write' len=32 type='asc' dur=5         NA       Inf %           NA   NA    NA
http/something.js method='write' len=32 type='asc' dur=5                   NA       NaN %           NA   NA    NA
```